MotionMark/resources/statistics.js

```diff
@@ -387,10 +383,8 @@ Regression = Utilities.createClass(
             n2: n2_best
         };
     }
-});
 
-Utilities.extendObject(Regression, {
```

Comment:
`Bootstrap` can be a class by itself. It takes the `samples` and `confidencePercentage` and uses the `Regression` to calculate the estimates. Using the estimates, Bootstrap can provide method for the `mean`, `median`, `confidence`, etc.

Reply:
Will do separately,